docker/Dockerfile

```diff
@@ -1,11 +1,13 @@
-FROM registry.access.redhat.com/ubi9/go-toolset:1.18.9-14
+FROM registry.access.redhat.com/ubi9/go-toolset:1.19.13-4.1697647145
 
-COPY . .
+COPY go.mod ./
```

Comment:
I would advise we not adopt this approach. Sometime early this year when testing out projects ppl brought during Burr's testing initiative, we found that doing a `COPY . .` had more success with random projects that just copying go files *.go and go.mod file.

Reply:
@maysunfaisal @kim-tsao I've reverted the update of the `dockerifle`. This was a potential workaround to fix the error:

```
 > [4/4] RUN go build -o ./main:
0.375 error obtaining VCS status: exit status 128
0.375   Use -buildvcs=false to disable VCS stamping.
------
Dockerfile:6
--------------------
   4 |     RUN go mod download
   5 |     
   6 | >>> RUN go build -o ./main
   7 |     
   8 |     ENV PORT 8081
--------------------
ERROR: failed to solve: process "/bin/sh -c go build -o ./main" did not complete successfully: exit code: 1
```

This can be reproduced if you try to build locally the dockerfile image.

Reply:
I've update the `go build` command with the flag `-buildvcs=false`. This is also done here: https://github.com/devfile/api/blob/main/build-generator.sh#L38